### PR TITLE
ImportC

### PR DESCRIPTION
**Pull request by al1-ce**
Pretty much finished it, example/Test complains about non-existing OnReady, but overall should work

### CODE REVIEW

File: examples/asteroids/project/asteroids.gdextension [hunk 1]
```diff
@@ -4,5 +4,5 @@ entry_symbol = "asteroids_gdextension_entry"
 
 [libraries]
 
-linux.64 = "libgodot-dlang_asteroids.so"
+linux.64 = "libasteroids.so"
```
Comment (Superbelko) [inline examples/asteroids/project/asteroids.gdextension:7]: Fix this pls, revert it back and instead update the dub config.

File: examples/test/project/Main.tscn [hunk 2]
```diff
@@ -8,17 +8,15 @@ script = ExtResource("1_nbra5")
 [node name="Panel" type="Panel" parent="."]
 
 [node name="Label" type="TestD" parent="Panel"]
-number = 24
-long_node_path = NodePath("One/Looooong/Incredibly/Unbroken/Node/Path/Label")
```
Comment (Superbelko) [inline examples/test/project/Main.tscn:12]: This was probably due to broken importc before, this change will probably should be reverted

File: examples/test/project/godot-d_test.gdextension [hunk 3]
```diff
@@ -4,5 +4,5 @@ entry_symbol = "test_gdextension_entry"
 
 [libraries]
 
-linux.64 = "libgodot-dlang_test.so"
+linux.64 = "libtest.so"
```
Comment (Superbelko) [inline examples/test/project/godot-d_test.gdextension:7]: same as asteroids, fix dub config

**Review by Superbelko — COMMENTED** [review]
Ok, but fix library names, also check the "test" project if it still works in play mode.
When you going to merge this please squash it in a single commit.

File: examples/test/project/Main.tscn (hunk 2 above)
Comment (al1-ce) [reply examples/test/project/Main.tscn:12]: Strangely enough it is not problem of importc coz godot does it by itself
Tried with both bindings and importc version, gonna need to be a bit more investigation

**Pull request merged**